### ISSUE TITLE
Screening for offensive vernacular names

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/tree-build#51]
A science educator wrote to me about Nasutitermes graveolus which had a pretty offensive vernacular name.

I deleted the name from our production site, but there will be a chance it reappears with tree rebuilds.

Whilst I don't think we can do anything about offensive scientific names where they exist, it is probably worth screening the vernacular name data for known offensive terms and just not including those names in the DB.  I think a script for this could be included in the tree building pipeline. It's more important that the site be considered safe from offensive terms than comprehensive.

**Comment by lentinj** [comment]
[EOL don't have any common names listed](https://eol.org/pages/3836838/names), so presumably the problem would have evaporated naturally (assuming I've got the source of common names right, wikipedia doesn't have a page for this termite).

But this isn't too different to the more unsavoury images that can come back from EoL (roadkill images, whilst useful for identification, isn't quite what we're going for). Ideally we'd feed this back to EoL rather than have our own layer, but IIRC Yan has tried and failed to interest them in a rating layer.

**Comment by hyanwong** [comment]
I don't think we should be in the business of sanitising names. We would hope that these just get sanitised for us by the process of scientific/vernacular revision (e.g. people deleting this from wikipedia)

I don't think this one will reappear on OneZoom, either through EoL or Wikidata. Indeed, it seems increasingly likely that we will aim to get names and a decent image from Wikidata, where it exists (see https://github.com/OneZoom/tree-build/issues/38). There seems to be neither vernaculars nor an image in this case: https://www.wikidata.org/wiki/Q110542108

**Comment by lentinj** [comment]
Is it worth having an FAQ entry of some sorts on this? Where we get our data from / who to report it to?

**Comment by hyanwong** [comment]
Yes, a FAQ (with contact details) is a good idea. Maybe once we have written that we can close this issue.

**Comment by hyanwong** [comment]
Topical! See https://www.nature.com/articles/d41586-024-02337-1
